src/radical/pilot/pmgr/launching/base.py

```diff
@@ -1081,6 +1070,7 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand, tar_name):
         jd_dict.wall_time_limit       = runtime
         jd_dict.queue                 = queue
         jd_dict.candidate_hosts       = candidate_hosts
+        jd_dict.file_transfer         = list()
```

Comment:
just a note that this attribute is not used anywhere else - we keep it just for legacy?

Reply:
Yes.